src/DataModel.php

```diff
@@ -76,9 +76,6 @@ public function __construct($source, string $primaryKey)
 			if ($driver instanceof Dibi\Drivers\OdbcDriver) {
 				$source = new DibiFluentMssqlDataSource($source, $primaryKey);
 
-			} elseif ($driver instanceof Dibi\Drivers\MsSqlDriver) {
-				$source = new DibiFluentMssqlDataSource($source, $primaryKey);
-
```

Comment:
Why is this code removed?

Reply:
https://github.com/dg/dibi/releases/tag/v4.0.0 mssqldriver was removed in dibi 4.0.0

Reply:
Then is should be maybe replaced with SQLSRV driver? (And tested if replacing)

Reply:
We can't just drop this support, it would be a BC Break.

Reply:
Condition for SqlsrvDriver is few lines below, so I think its fine.

I added class_exists check, hopefully will be sufficient for phpstan :-)